debian-arm64: Use arm64v8/debian:trixie as base

jsuhaas22 opened the pull request:
Currently, we are using arm64v8/debian:trixie-slim as the base. But this image seems to be unavailable, as a result of which our workflow is failing.

Therefore use, in its stead, arm64v8/debian:trixie. This layer seems to be available.

jsuhaas22 committed:
workflow: docker-image: Specify platform for debian-arm64

Specify --platform=linux/arm64 for building debian-arm64. Without this,
it is unable to pull from the debian:trixie-slim image, since we are
building for arm64 image for amd64 qemu system.

Since we have only 2 Dockerfiles for now and the foreseeable future,
replace the loop with 2 individual commands to accomplish this.

Signed-off-by: Suhaas Joshi <s-joshi@ti.com>

babldev commented:
cc @cshilwant 

I noticed the Ubuntu Docker image is out of date so it would be good to fix the image deploy pipeline.

babldev commented:
I see trixie slim here:
https://hub.docker.com/layers/arm64v8/debian/trixie-slim/images/sha256-c4c6148422aad5a8ac8690eb14149b72d34f2b5eae614d06e2f189775882be36

Maybe unavailability was temporary?

jsuhaas22 committed:
debian-arm64: Add dependencies for ti-apps-launcher

ti-apps-launcher [0] is a Qt-based application built as a demo for TI
platforms. To provide an easier way to compile a binary from it, include
Qt dependencies in the image.

[0] https://github.com/texasinstruments/ti-apps-launcher

Signed-off-by: Suhaas Joshi <s-joshi@ti.com>

cshilwant commented:
@babldev Thanks for reporting the issue. The image deployment workflow is fixed with this PR.
